scripts/ELK/templates/metricbeat.yml

```diff
@@ -91,7 +91,7 @@ setup.kibana:
 # ---------------------------- Elasticsearch Output ----------------------------
 output.elasticsearch:
   # Array of hosts to connect to.
-  hosts: ["<ELASTIC-MACHINE-PUBLIC-IP>:9200"] # eg: ["134.209.25.17:9200"]
+  hosts: ["188.166.96.224:9200"] # eg: ["134.209.25.17:9200"]
```

Comment:
This one seems we're not picking for ourselves. Will we have that same IP "always" there?

Reply:
a crap. removed!